single_line_class.rb

```diff
@@ -0,0 +1,27 @@
+#Prefer a single-line format for class definitions with no body
+class FooError < StandardError
+end
+
+class Test
+	# Avoid single-line methods.
```

Comment:
__[rubocop]__
* Tab detected. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)
* Incorrect indentation detected (column 1 instead of 4).